packs/slack/sensors/slack_sensor.py

```diff
@@ -35,7 +36,8 @@ def setup(self):
             msg = 'Failed to connect to the Slack API. Invalid token?'
             raise Exception(msg)
 
-        self._populate_cache(user_data=data['users'], channel_data=data['channels'])
+        self._populate_cache(user_data=self._api_call('users.list'),
```

Comment:
I see and yeah, I think some of the recent API changes were also the reason for `is_admin` stuff breaking.

And yeah, as long as we seed it with all the users at the beginning I'm fine with it.